src/userStatusOverview/types.ts

```diff
@@ -1,26 +1,36 @@
+export enum UserStatusOverviewStatus {
+  Offline,
+  Working,
+  Break,
+  Leave,
+  CoreTimeViolation = 5,
+}
+
 export type UserStatusOverview = {
-  user_id?: number;
+  user_id: number;
   /**
    * Time tracking id of a running task
    */
-  time_tracking_id?: number;
-  time_tracking_start_time?: string;
-  time_tracking_start_time_timezone_id?: string;
-  time_tracking_task_id?: number;
-  time_tracking_is_nonworking?: boolean;
+  time_tracking_id: number;
+  time_tracking_start_time: string | null;
```

Comment:
I'd prefer if we could use optional instead of null, but I'm not sure how we could do that. eg. if we should change null to undefined in responseHandler

Reply:
would also prefer optional instead of null.

Reply:
as I remember we talked that we will clarify types for fields, so if we switch to optional then it means that we allow field to be undefined

Reply:
We'd have to look into how to do it in a separate story